### PR TITLE
Fix Broken Links

### PR DESCRIPTION
**Pull request by scetron**
Fix broken links that have been moved or updated in Nautobot 2.0/stable.

### CODE REVIEW

**Review by glennmatthews — APPROVED** [review]
Thank you!

File: html/index.html [hunk 1]
```diff
@@ -103,7 +103,7 @@ <h1 class="align-text-center">
                         href="https://www.youtube.com/playlist?list=PLjA0bhxgryJ2Ts4GJMDA-tPzVWEncv4pb"
                         class="nav-link">Videos</a></li>
                 <li class="nav-item"><a target="_blank"
-                        href="https://www.networktocode.com/academy/enterprise/nautobot-package-iv/"
+                        href="https://networktocode.com/how-we-do-it/enablement-adoption/network-automation-academy/"
```
Comment (glennmatthews) [inline html/index.html]: Should this be the same link as we have at line 320 for the footer?

Comment (scetron) [reply html/index.html]: Yes, that is correct. Thanks for the catch. They should both be changed now.

Comment (cmsirbu) [inline html/index.html]:
```suggestion
                        href="https://networktocode.com/course/package-iv-nautobot/"
```

Comment (cmsirbu) [reply html/index.html]: The issue here was the `www` prefix - we want to link to the Nautobot training specifically.

File: html/index.html [hunk 2]
```diff
@@ -317,7 +317,7 @@ <h5>Products</h5>
                                 href="https://networktocode.com/nautobot/nautobot-cloud/">Nautobot Cloud</a>
                         </li>
                         <li class="mb-1"><a class="link-light text-decoration-none"
-                                href="https://networktocode.com/course/package-iv-nautobot/">Nautobot
+                                href="https://networktocode.com/how-we-do-it/enablement-adoption/network-automation-academy/">Nautobot
```
Comment (cmsirbu) [inline html/index.html]:
```suggestion
                                href="https://networktocode.com/course/package-iv-nautobot/">Nautobot
```

Comment (cmsirbu) [reply html/index.html]: This link was working.

**Review by cmsirbu — CHANGES_REQUESTED** [review]
Thank you for fixing the broken links! I fixed and reverted the training link changes.

**Review by cmsirbu — APPROVED** [review]
LGTM, thanks!

**Pull request merged**